ui/app/styles/pages/auth-page.scss

```diff
@@ -73,10 +73,9 @@
       margin-bottom: scale.$lg-2;
     }
   }
-  .oidc-callback {
-    .spinner {
-      font-size: 48px;
-      margin: 24px 80px;
-    }
+
+  .button-align {
```

Comment:
**[question, non-blocking]**

Where is this class used?

Reply:
Must have deleted this in my template wrangling, will fix and report back

Reply:
Added this back, with flexbox gap